package.json

```diff
@@ -54,7 +54,7 @@
     "@testing-library/react-hooks": "^8.0.0",
     "@types/jest": "^27.5.1",
     "@types/node": "^16.0.0",
-    "@types/react": "^17.0.0",
```

Comment:
shouldn't we remove `@testing-library/react-hooks`?

Reply:
[Same](https://github.com/croct-tech/plug-react/pull/178#discussion_r899998314)